spec/index.html

```diff
@@ -612,6 +624,49 @@ <h3>Version Labels</h3>
         </tr>
       </tbody>
     </table>
+
+    <p>For serializations supporting in-line version annouoncement,
+      the version announcement SHOULD be made early in the document
+      and certainly before serializing any feature depending on that version.</p>
+
+    <section id="server-considerations" class="informative">
+      <h4>Server considerations</h4>
+      <p>When serializing a graph or dataset which uses features incompatible
+        with a requested version, servers can consider different alternatives:</p>
+
+      <ol>
+        <li>Downgrade <a>Triple Terms</a> by using an algorithm such
```

Comment:
To me, "downgrade triple terms" does not clearly convey the idea to actually eliminate the triple terms; it sounds more like somehow changing triple terms in order to "downgrade" them. Therefore, I propose to modify this part of the sentence as follows.
```suggestion
        <li>Eliminate <a>triple terms</a> by using an algorithm such
```